apps/web/utils/middleware.ts

```
@@ -14,12 +14,15 @@ import { env } from "@/env";
 import { posthogCaptureEvent } from "@/utils/posthog";
 import { auth } from "@/app/api/auth/[...nextauth]/auth";
 
-export type NextHandler = (
+type NextHandler<TContext extends Record<string, any>, TResponse = void> = (
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)